.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "es5"
```

Comment:
Thanks for doing this, I assume splitting it out allows us to have central configuration that can be used by a build step 👍

Reply:
Yes, this makes the prettier config explicit and not only available to vscode.